apps/src/pcd_select_object_plane.cpp

```diff
@@ -637,10 +637,8 @@ main (int argc, char** argv)
     print_info ("Ideally, need an input file, and three output PCD files, e.g., object.pcd, plane.pcd, rest.pcd\n");
     return (-1);
   }
-  
-  string object_file = "object.pcd", plane_file = "plane.pcd", rest_file = "rest.pcd";
+
   if (p_file_indices.size () >= 4)
```

Comment:
Line 637 suggests that the logic might be wrong, and removing the file name variables might be the wrong way to resolve the error. The variables are unused because some functionality is missing